new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,11 +54,84 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info(String.format("Got a new namespace called %s", namespace));
 
+    // create a persistent volume claim
+    final String storageClassName = domainUID + "-weblogic-domain-storage-class";
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+    HashMap<String, String> labels = new HashMap();
+    labels.put("weblogic.resourceVersion", "domain-v2");
+    labels.put("weblogic.domainUID", domainUID); // label it with domain uid
+    final List<String> accessMode = new ArrayList();
+    accessMode.add("ReadWriteMany"); // access mode of the persistent volume claim
+    final String storage = "10Gi";
+    final String volumeMode = "Filesystem";
+    final String volumeName = domainUID + "-weblogic-pv"; // volume name
+    final String pvPath = System.getProperty("java.io.tmpdir") + domainUID + "-persistentVolume";
+    final String persistentVolumeReclaimPolicy = "Recycle"; // one of Recycle, Retain, Delete
+    Quantity maxClaims = Quantity.fromString(storage);
+    Map<String, Quantity> capacity = new HashMap<>();
```

Comment:
Same. V1PersistentVolumeSpec has putCapacityItem()

Reply:
fixed it.